pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -141,6 +145,18 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	// Don't modify the informer's copy.
 	tr := original.DeepCopy()
 
+	// TODO(aaron-prindle) verify that build does not have to exist to create a tracker
+	buildRef := corev1.ObjectReference{
+		APIVersion: "build.knative.dev/v1alpha1",
+		Kind:       "Build",
+		Namespace:  tr.Namespace,
+		Name:       tr.Name,
+	}
+	if err := c.tracker.Track(buildRef, tr); err != nil {
+		logger.Errorf("build %s for taskrun %s: %v", buildRef, tr.Name, err)
```

Comment:
- [ ] could the error say something about what it was trying to do? e.g. something like `failed to create tracker for build %s for taskrun %s: %v`